flink-core/src/main/java/org/apache/flink/util/AbstractID.java

```diff
@@ -46,7 +46,7 @@
 	/** The lower part of the actual ID */
 	protected final long lowerPart;
 
-	/** The memoized value returned by toString() */
+	/** The memorized value returned by toString() */
```

Comment:
Oh, hey, that was me. I was just thinking about this the other day and how the JobManager used to bottleneck for large jobs when logging and converting these `AbstractID` to string. Anyway, this original should be the correct spelling: https://en.wikipedia.org/wiki/Memoization